integration_tests/src/main/python/orc_test.py

```diff
@@ -486,4 +486,47 @@ def test_orc_read_with_corrupt_files(spark_tmp_path, reader_confs, v1_enabled_li
 
     assert_gpu_and_cpu_are_equal_collect(
             lambda spark : spark.read.orc([first_data_path, second_data_path, third_data_path]),
-            conf=all_confs)
\ No newline at end of file
+            conf=all_confs)
+
+conf_for_orc_aggregate_pushdown = {
+    "spark.sql.orc.aggregatePushdown": "true", 
+    "spark.sql.sources.useV1SourceList": ""
+}
+
+@pytest.mark.skipif(is_before_spark_330(), reason='Aggregate push down on ORC is a new feature of Spark 330')
+@allow_non_gpu(any = True)
+def test_orc_scan_with_aggregation_pushdown_fallback(spark_tmp_path):
+    """
+    The aggregation will be pushed down in this test, so we should fallback to CPU
+    """
+    data_path = spark_tmp_path + '/pushdown.orc'
+
+    def do_orc_scan(spark):
+        df = spark.read.orc(data_path).selectExpr("count(p)")
+        return df
+    
+    with_cpu_session(lambda spark : spark.range(10).selectExpr("id", "id % 3 as p").write.partitionBy("p").mode("overwrite").orc(data_path))
+
+    assert_cpu_and_gpu_are_equal_collect_with_capture(
+        do_orc_scan,
+        exist_classes= "BatchScanExec",
+        non_exist_classes= "GpuBatchScanExec",
+        conf = conf_for_orc_aggregate_pushdown)
+
+@pytest.mark.skipif(is_before_spark_330(), reason='Aggregate push down on ORC is a new feature of Spark 330')
+def test_orc_scan_without_aggregation_pushdown_not_fallback(spark_tmp_path):
+    """
+    No aggregation will be pushed down in this test, so we should not fallback to CPU
+    """
+    data_path = spark_tmp_path + "/pushdown.orc"
+
+    def do_orc_scan(spark):
+        df = spark.read.orc(data_path).selectExpr("Max(p)")
```

Comment:
Why is max not pushed down but count is?

Reply:
I was looking to [Spark-PR-34298](https://github.com/apache/spark/pull/34298). 
AFAIU, Min/Max don't push down partition column. Only Count does.

Reply:
That is fine, then add a comment about it and update the name of this issues so only Count is included. Spark messed up by keeping the name of the issue and PR the same.

Reply:
I updated the test to cover all the aggregates (min/max/count) Vs. (None/- partition columns).